package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "merge-md",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

```suggestion
  "version": "1.1.0",
```
Your PR is a backwards compatible feature, so let's bump the minor number.